api/mysagw/identity/tests/test_membership_views.py

```diff
@@ -136,6 +137,28 @@ def test_membership_role_create(db, client, expected_status):
     }
 
 
+@pytest.mark.parametrize(
+    "title",
+    [
+        {"de": ""},
+        {"de": None},
+        {"en": "Foo"},
+        {},
```

Comment:
I suggest adding a positive test case as well, where the request is accepted

Reply:
There already is one, right above the new one. The existing also asserts permissions, that's why I've decided to not bloat it.

Reply:
Fair, alright then :+1: